plugins/by-name/clangd-extensions/default.nix

```
@@ -76,18 +76,30 @@ lib.nixvim.plugins.mkNeovimPlugin {
 
   extraConfig = cfg: {
     warnings = lib.nixvim.mkWarnings "plugins.clangd-extensions" {
-      when = !config.plugins.lsp.enable;
+      when = !((config.plugins.lsp.enable or false) || (config.lsp.servers.clangd.enable or false));
```

We still shouldn't need `or false` for either of these.

`or` use useful when an attr is allowed to be missing; the module system guarantees that all declared options will have a corresponding `config` attr.

Even if the value is undefined (which is impossible for options that have a default), that is implemented as the attr being present but defined as a `throw` expression. IIRC `throw "Option foo.bar.baz used but not defined"`.

Therefore `?` and `or` are only really useful in the module system, when you are dealing with the sub-attrs of some freeform option; e.g. an `attrsOf` option.

```suggestion
      when = !(config.plugins.lsp.enable || config.lsp.servers.clangd.enable);
```

---

Thanks, sorry I thought I had already fixed this.